Template/deviceConfiguration.json

```diff
@@ -28,13 +28,13 @@
 							"OptimizeForPerformance": {
 								"value": "false"
 							},
-              "mqttSettings__enabled": {
-                "value": "false"
-              },
-              "AuthenticationMode": {
-                "value": "CloudAndScope"
-              }
-			},
+							"mqttSettings__enabled": {
+								"value": "false"
+							},
+							"AuthenticationMode": {
```

Comment:
As discussed yesterday if we plan to deply in direct mode (enable_gateway = false) we can remove this setting. If we don't we probably need to also disable nestededge

Reply:
You are right, I will update accordingly this section as soon as #948 gets merged.

Reply:
Done